Automattic/vip-go-mu-plugins
# Update Rewrite Rules Inspector plugin to latest version to resolve deprecated warnings

## Description

Navigating to VIP > Rewrite Rules throws the below errors locally:

```
Notice: screen_icon is deprecated since version 3.8.0 with no alternative available. in /Users/Shared/IH/Integrated/wp-includes/functions.php on line 4332

Notice: get_screen_icon is deprecated since version 3.8.0 with no alternative available. in /Users/Shared/IH/Integrated/wp-includes/functions.php on line 4332
```

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR and ensure `WP_DEBUG` is set to `true`.
1. Go to VIP > Rewrite Rules
1. Verify that the below warnings are no longer appearing:
<img width="1127" alt="Screen Shot 2019-04-15 at 4 41 21 PM" src="https://user-images.githubusercontent.com/16962021/56170238-054e4d80-5f9e-11e9-912f-984c7fb66d44.png">
